BREEZE.rst

```diff
@@ -137,131 +131,102 @@ You can always stop it via:
     ./breeze --stop-environment
 
 
-Using the Airflow Breeze environment for testing
-================================================
-
-Setting up autocomplete
+Setting Up Autocompletion
 -----------------------
 
-The ``breeze`` command comes with built-in bash/zsh autocomplete for its flags. When you start typing
-the command you can use <TAB> to show all the available switches
-nd to get autocompletion on typical values of parameters that you can use.
+The ``breeze`` command comes with a built-in bash/zsh autocomplete option for its flags. When you start typing
+the command, you can use <TAB> to show all the available switches and get autocompletion on typical values of parameters that you can use.
 
-You can setup auto-complete automatically by running:
+You can set up the autocomplete option automatically by running:
 
 .. code-block:: bash
 
    ./breeze --setup-autocomplete
 
-You get autocomplete working when you re-enter the shell.
+You get the autocompletion working when you re-enter the shell.
 
 Zsh autocompletion is currently limited to only autocomplete flags. Bash autocompletion also completes
-flag values (for example python version or static check name).
-
-Entering the environment
-------------------------
-
-You enter the integration test environment by running the ``./breeze`` script.
-
-What happens next is the appropriate docker images are pulled, local sources are used to build local version
-of the image and you are dropped into bash shell of the airflow container -
-with all necessary dependencies started up. Note that the first run (per python) might take up to 10 minutes
-on a fast connection to start. Subsequent runs should be much faster.
+flag values (for example, Python version or static check name).
 
-.. code-block:: bash
-
-   ./breeze
-
-Once you enter the environment you are dropped into bash shell and you can run tests immediately.
-
-Choosing environment
+Choosing Environment
```

Comment:
you are not talking here about choosing env. I remember you used to have a list of possible environments in this doc. But now you are talking only about docker. why choosing then if there is no choice?

Reply:
It's all about choosing Python + Backend while entering Breeze rather than whole environment.